streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -425,6 +427,9 @@ public void flushInternal() {
     @Override
     public void close() {
         flush();
+        options.dispose();
+        wOptions.dispose();
+        fOptions.dispose();
```

Comment:
I think you want to null out these fields after you call `dispose` because using them after this will lead to undefined behaviour (a NPE is preferable in case of misuse).